### PR TITLE
Add separate CSS for index page browse lists

### PR DESCRIPTION
**Pull request by tombye**
### With old browse list

homepage_existing_browse_list

### With new browse list

homepage_new_browse_list

### With old browse list

g-cloud_index_existing_browse_list

### With new browse list

g-cloud_index_new_browse_list

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -104,7 +104,7 @@ <h2 id="supplier-message-heading">Sell services</h2>
           {% if logged_in %}
             <p>
               <a href="/suppliers" class="top-level-link">
-                <span>View your services and account details</span>
+                View your services and account details
```
Comment (TheDoubleK) [inline app/templates/index.html]: I think the `{% if logged_in %}` condition here needs to be more specific - it should only show this link/sidebar if logged in as a supplier, not as a buyer.

Comment (TheDoubleK) [inline app/templates/index.html]: We might want a different link for logged-in buyer users, to take them to their buyer dashboard `/buyers` instead.

Comment (tombye) [inline app/templates/index.html]: That's a good point. @ralph-hawkins has a design for a link for buyers too so I'll put it in and make the condition more specific.

**Comment by tombye** [comment]
@TheDoubleK I've made some changes in line with your comments:

1. `index.html` now gets sent the `logged_in_user` variable which is the role or `None`
2. the link to the supplier dashboard is now tied to the logged in user being a supplier
3. the link to the buyer dashboard (which was already in the main browse-list) now only appears if a       buyer user is logged in
4. I changed the text in the buyer dashboard link to match that provided by @ralph-hawkins

File: app/main/views/marketplace.py [hunk 2]
```diff
@@ -42,11 +42,15 @@ def index():
             "framework {} status {}".format(framework.get('slug'), framework.get('status')))
         abort(500)
 
+    if current_user.is_authenticated():
+        logged_in_user = current_user.role
```
Comment (TheDoubleK) [inline app/main/views/marketplace.py]: I don't think you need to pass anything into the template at all here, as templates have access to `current_user`.

So, e.g. in _proposition_header.html we have `{% if current_user.email_address %}` and for the new links we can do in the template: `{% if current_user.role == 'supplier' %}` with no need for an additional parameter.

**Comment by tombye** [comment]
@TheDoubleK I've made some changes so `current_user` is used and it's not necessary to mock it in the tests.

**Comment by TheDoubleK** [comment]
:+1:  :shipit:  :+1:

**Pull request merged**